js/interface.js

```diff
@@ -73,7 +73,7 @@ var definitionEditor = CodeMirror.fromTextArea($('#definition')[0], {
 
 var emptyColumnNameRegex = /^Column\s\([0-9]+\)$/;
 
-Fliplet.App.Tokens.get({appId: widgetData.appId, query: {type: 'integrationToken'}}).then(function(tokens) {
+Fliplet.App.Tokens.get({ appId: widgetData.appId, query: { type: 'integrationToken', order:'createdAt', direction:'DESC' } }).then(function(tokens) {
```

Comment:
As per ESLint, space after colon. Can you confirm you set up ESLint?

Reply:
Hi @squallstar 
currently in my package.json file eslint-config-fliplet version  is **"^1.0.8"** . Due to this I'm not getting eslint issue over there. After changing eslint-config-fliplet version to **"^0.8.0"** Now I'm able to getting the eslint issue there .
Please see the below screenshot:
![image](https://user-images.githubusercontent.com/102005879/190383642-c209c590-c785-42c3-b12f-414cfb0a29fa.png)